inst/pages/04_containers.qmd

```diff
@@ -740,3 +740,39 @@ Conversion is possible between other data formats. Interested readers can refer
 * [makeTreeSEFromBiom](https://microbiome.github.io/mia/reference/makeTreeSEFromBiom.html)  
 * [importMetaPhlAn](https://microbiome.github.io/mia/reference/importMetaPhlAn.html)  
 * [importQZA](https://microbiome.github.io/mia/reference/importQIIME2.html)
+
+## Exporting data container {#sec-exporting-data-container}
+
+### Export TreeSummarizedExperiment
+
+Exporting a `TreeSE` data container can be done using *feather* package. TSE object has to be converted into a dataframe (`DataFrame`). The output file is a .feather file, which can be imported in other languages such as Julia or Python. For information, have a look at the [FeatherFile](https://juliapackages.com/p/featherfiles) Julia package and [feather-format](https://pypi.org/project/feather-format/) Python library.
```

Comment:
Does also data.frame object work? DataFrame and data.frame are different. DF inherits data.frame. One option is also basic csv file (you could add that).